packages/line/src/Line.js

```diff
@@ -62,6 +69,9 @@ const Line = props => {
         pointLabelFormat,
         pointLabelYOffset,
 
+        defs,
```

Comment:
Do we need to add defs and fill to the Type definitions? i.e.

`export interface LineSvgProps extends LineProps, MotionProps, **SvgDefsAndFill<Datum>** { ... }`

Reply:
Thanks, good catch!